model/kubernetes/kubernetes_node.rb

```diff
@@ -10,6 +10,25 @@ class KubernetesNode < Sequel::Model
 
   plugin ResourceMethods
   plugin SemaphoreMethods, :destroy
+
+  def sshable
+    vm.sshable
+  end
+
+  def worker_billing_records
```

Comment:
nit: why not have a single billing_record function that returns CP billing record or worker billing record depending on whether there's a nodepool associated or not?